front/src/components/post-button-and-function/postButton.component.jsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { postFunction } from "./postFunction";
```

Comment:
there is no postFunction exported in postFunction.js :)

Reply:
It is, i just use export const without default instead of export default at the end
more info here => https://stackoverflow.com/questions/33611812/export-const-vs-export-default-in-es6